.drone.star

```diff
@@ -1,23 +1,23 @@
 def main(ctx):
   versions = [
     {
-      'value': '10.4.1-rc2',
-      'qa': 'https://download.owncloud.org/community/testing/owncloud-10.4.1RC2-qa.tar.bz2',
-      'tarball': 'https://download.owncloud.org/community/testing/owncloud-10.4.1RC2.tar.bz2',
-      'tarball_sha': '055482c83bf02419e5f2fe1c1e8d0864b200e23ae29409b6d2412e02d64dadf3',
-      'ldap': 'https://marketplace.owncloud.com/api/v1/apps/user_ldap/0.15.0',
-      'ldap_sha': '742703c66a8a112636c30aa500d7b4fdff670bac179fe3ae658a2b4016e5847f',
+      'value': '10.4.1',
+      'qa': 'https://download.owncloud.org/community/owncloud-10.4.1-qa.tar.bz2',
+      'tarball': 'https://download.owncloud.org/community/owncloud-10.4.1.tar.bz2',
+      'tarball_sha': '63f32048225c6bc4534c6757e8beee65fc845a35126899e85d787a3ba4073d48',
+      'ldap': 'https://marketplace.owncloud.com/api/v1/apps/user_ldap/0.15.1',
+      'ldap_sha': '1e34bb56850ac93c8625809247a6d7b23113eda53c456c7560dda1f58f42ab93',
       'php': '7.3',
       'base': 'v19.10',
       'tags': [],
     },
     {
       'value': 'latest',
-      'qa': 'https://download.owncloud.org/community/owncloud-10.4.0-qa.tar.bz2',
-      'tarball': 'https://download.owncloud.org/community/owncloud-10.4.0.tar.bz2',
-      'tarball_sha': 'ec6ac9a507f4196453da4526f44cc079b96f7e7863bafe4eb6ba373962907bb6',
-      'ldap': 'https://marketplace.owncloud.com/api/v1/apps/user_ldap/0.15.0',
-      'ldap_sha': '742703c66a8a112636c30aa500d7b4fdff670bac179fe3ae658a2b4016e5847f',
+      'qa': 'https://download.owncloud.org/community/owncloud-10.4.1-qa.tar.bz2',
+      'tarball': 'https://download.owncloud.org/community/owncloud-10.4.1.tar.bz2',
+      'tarball_sha': '63f32048225c6bc4534c6757e8beee65fc845a35126899e85d787a3ba4073d48',
+      'ldap': 'https://marketplace.owncloud.com/api/v1/apps/user_ldap/0.15.1',
+      'ldap_sha': '1e34bb56850ac93c8625809247a6d7b23113eda53c456c7560dda1f58f42ab93',
       'php': '7.3',
       'behat_version': 'behat-10.4.0',
```

Comment:
need to remove this line - for 10.4.1 we do not need to fetch any special behat composer.json

Reply:
is it the latest? Latest needs to be specified use `v10.4.1`

Reply:
yes, IMO that is correct, because the starlark code won't know magically when it is processing `latest`
```suggestion
      'behat_version': 'v10.4.1',
```